Index.html

```diff
@@ -55,6 +56,161 @@ <h1 id="primary_text">
+        
+        <ul class="projects_tag_box">
+          <li class = "projects_box">html</li>
+          <li class = "projects_box">bootstrap</li>
+          <li class = "projects_box">Ruby</li>
+        </ul>
+    
+        <button type="button" class="projects_button button_interactions" >See Project</button>
+      </div>
+
+      <div class="projects">
+        <h1 id="projects_header">Profesional Art Printing Data</h1>
+        <p class="projects_description">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+        
+        <ul class="projects_tag_box">
+          <li class = "projects_box">html</li>
+          <li class = "projects_box">bootstrap</li>
+          <li class = "projects_box">Ruby</li>
+        </ul>
+    
+        <button type="button" class="projects_button button_interactions" >See Project</button>
+      </div>
+
+      <div class="projects">
+        <h1 id="projects_header">Profesional Art Printing Data</h1>
+        <p class="projects_description">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+        
+        <ul class="projects_tag_box">
+          <li class = "projects_box">html</li>
+          <li class = "projects_box">bootstrap</li>
+          <li class = "projects_box">Ruby</li>
+        </ul>
+    
+        <button type="button" class="projects_button button_interactions" >See Project</button>
+      </div>
+      
+    </div>
+
+    </section>
+
+    <!-- About Me Section -->
+
+    <section class="about_me">
+      <h1 id="about_me_heading">About me</h1>
+      <p id="about_me_description">
+        Hello I’m a software developer! I can help you build a product ,
+        feature or website Look through some of my work and experience!
+        If you like what you seeand have a project you need coded,
+        don’t hestiate to contact me.
+      </p>
+      <button type="button" id="about_me_button" class="button_interactions">Get My Resume</button>
+      <img src="Sources/Vector 4.png" id="break_line2">
+    </section>
+
+      <div class="my_skills">
+      
+      <h3 class="list_header">
+      <img src="Sources/Rectangle 55.png"></img>
+      Language</h3>
+      <ul class="skills_list">
+        <li>Javascript</li>
+        <li>Ruby</li>
+        <li>Html</li>
+        <li>Css</li>
+      </ul>
+
+      <h3 class="list_header">
+      <img src="Sources/Rectangle 56.png"></img>
+      Frameworks</h3>
+      <ul class="skills_list">
+        <li>Bootstrap</li>
+        <li>Ruby on Rails</li>
+        <li>RSpec</li>
+        <li>CapyBara</li>
+        <li>Selenium</li>
+      </ul>
+
+      <h3 class="list_header">
+      <img src="Sources/Rectangle 57.png">
+      </img>Skills</h3>
+      <ul class="skills_list">
+        <li>Codekit</li>
+        <li>Github</li>
```

Comment:
- [ ] Great job on your good design 👏. However, from the markup design, you are expected to implement it as the `nested list`. The technology categories(e.g: `Languages`, `Framework`, etc) are supposed to be in `<ul>` where each is in `<li>` and each technology(e.g: `HTML`, `CSS`, etc) should be in <ul> each in `<li>`. Well, it may sound confusing and you may not be getting how you are going to implement it. Here is a very good article that will help you get the point 👉 [HERE](https://www.webucator.com/article/how-to-properly-nest-lists-in-html/)

Reply:
Sir @BertrandConxy ,

Your requested requirements have been modified and a review is requested on the dashboard.
Thank you for being patient.